mirror-service-x/src/main/java/org/hyades/vulnmirror/datasource/osv/OsvConfig.java

```diff
@@ -0,0 +1,14 @@
+package org.hyades.vulnmirror.datasource.osv;
+
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
+
+import java.util.Optional;
+
+@ConfigMapping(prefix = "mirror.osv")
```

Comment:
Should be `mirror.datasource.osv` to match the other datasources.

Reply:
fixed